tools/notification-configuration/notification-creator/NotificationConfigurator.cs

```diff
@@ -186,8 +187,18 @@ private async Task SyncTeamWithCodeOwnerFile(BuildDefinition pipeline, WebApiTea
 
                 // Get contents of CODEOWNERS
                 logger.LogInformation("Fetching CODEOWNERS file");
-                var managementUrl = new Uri(pipeline.Repository.Properties["manageUrl"]);
-                var codeOwnerEntries = await gitHubService.GetCodeownersFile(managementUrl);
+                var repoUrl = pipeline.Repository.Url;
```

Comment:
Can you use the real type here instead of var? It isn't obvious if this is a string or not.

Reply:
Done